rules-topics/01_introduction.md

```diff
@@ -108,3 +158,16 @@ The exact physical locations of these vary by camp site, although they are usual
 Don't be afraid to ask questions. Even in the middle of combat or intense role playing, don't be afraid to ask what something does or means. There are also many people with a decent grasp on the rules who can answer questions. In game, asking questions is just as useful, although may not always get you answers.
 
 It's just a game. We're all here to have fun. Try not to let any in game interactions affect your opinions of a person. Many people act in ways that they never would in real life. That's the joy of the event. It's not personal, they don't hate you, just your character. However, if you believe anything negative is being done out of game, or feel yourself getting frustrated, don't hesitate to talk to staff, or go out of game for a while to collect yourself.
+
+### Game Atmosphere
+
+Chronicles science, medicine, and technology in roughly in the 14th century (middle ages) so players are encouraged to act like, speak like, and generally behave like they are living in that time frame. We do of course allow players to use modern medicine but if possible, it should be done away from a large crowd to promote the in game atmosphere.
+
+**Religion**
+
+There are beings known as gods in Chronicles but if and how you worship them is up to you. No player will have to worship a god if they do not desire to. In order to not accidentally offend any religion players are encouraged to nor use religious symbols in costumes. Players should also avoid using gods, demons, or other non earthy entities name or religions while at a market fair.
```

Comment:
Suggestion here to separate player & character &: `There are beings known as gods in Chronicles, but how your character views or engages with them is up to the individual player.  No character is required to worship a god.`

`We strive to be inclusive and respectful at Chronicles and do not want anyone to be in the situation of giving offense to others.  Players are discouraged from using real world religious symbols or iconography with their costuming, garb, or props.  Similarly, in roleplay, players should also avoid use of or invocations of gods, demons, or other recognizable figures of real world religious or spiritual importance.`

Reply:
Thanks I like your wording better